### DISCUSSION TITLE
GDPR Compliance

### DISCUSSION THREAD

**Discussion by timothymalcham** [epicweb-dev/epic-stack#615] [Q&A] [answered]
Does anybody have any experience making their Epic Stack app GDPR-compliant? 

I'm trying to wade into this as I'll likely have EU customers, and I've never had to worry about this before. Cookies are used for auth/sessions, so I know they're essential for the Epic Stack, so I don't know how I'd allow a user to manage their cookies. If anyone has any pointers, that would be awesome! Thanks.

**Comment by kentcdodds** [comment, marked as answer]
I'm not a lawyer, but I'm pretty sure there's nothing in the epic stack that is not GDPR compliant. So as long as you don't add anything extra that breaks compliance you should be good.

**Reply by timothymalcham** [reply]
That's good to know, thank you @kentcdodds!
